1. *** Error in `python': double free or corruption (!prev):
Possible reason: list index out of range

2. Segmentfault
Possible reasons:
    + check pointer 
        + Did you have: self.thisptr = ... (should be "self.thisptr[0] or deref(self.thisptr)")
        + Did you delete the pointer twice or delete unallocated one?
    + Check input file name:
        Tc5b.Tc5b.nat.crd --> Tc5b.nat.crd (need to have more type-checking)

3. Getting elemenet from vector: return NoneType
    + Make sure to "return" something in __getitem__(self, idx) method
